source/composer.apib

```diff
@@ -6,6 +6,12 @@ The composition of API Blueprint is performed as it is provided by the [Matter C
 
 #### Input Media Types
 
+##### API Elements
+
+API Elements is supported, you can send an [API Elements API
+Category](http://api-elements.readthedocs.io/en/latest/element-definitions/#category-base-api-element)
+to the composer using the `application/vnd.refract+json` content type.
```

Comment:
We have decided to support both Parse Result and API Category. So, let's edit this description and have both transaction examples here.